__tests__/app.test.js

```diff
@@ -29,3 +29,40 @@ describe("GET /api/topics", () => {
         })
     })
 })
+
+describe("GET /api/articles/:article_id", () => {
+    test('returns article by id with correct properties', () => {
+        return request(app).get('/api/articles/2')
+        .expect(200)
+        .then(({body}) => {
+            const articles = body.article
+            articles.forEach((article) => {
```

Comment:
This task requires an object comes back, so body.article should be an object instead of an array of one object